x-pack/plugin/spatial/src/main/java/org/elasticsearch/xpack/spatial/SpatialPlugin.java

```diff
@@ -214,6 +217,9 @@ private static void registerCardinalityAggregator(ValuesSourceRegistry.Builder b
     @Override
     public void loadExtensions(ExtensionLoader loader) {
         // we only expect one vector tile extension that comes from the vector tile module.
-        loader.loadExtensions(VectorTileExtension.class).forEach(vectorTileExtension::set);
+        List<GeoFormatterFactory.FormatterFactory<Geometry>> formatterFactories = new ArrayList<>();
```

Comment:
In theory we can support more that one extension now but we are limiting to only one. I think this is good for now as we don't want external plugins to use this extension point for now.

Reply:
I think it should work with multiple extensions as it is right now, as long as they use different formatter names:
```
loader.loadExtensions(GeometryFormatterExtension.class)         // this should return a list of all extensions
    .stream()
    .map(GeometryFormatterExtension::getGeometryFormatterFactories) // this will extract lists of all factories
    .forEach(formatterFactories::addAll);                           // here we will merge all elements from all lists to a single array
geoFormatterFactory.set(new GeoFormatterFactory<>(formatterFactories)); // and here we should build a factory 
                                                                    // that should, theoretically, have all formatters in
```

Reply:
But only one plugin can add factories

Reply:
I had a chat offline with Igor and I was wrong as this method is called once for all extensions. Still it looks safe is anyone wants to register its own extension, the only downside is that it only applies to geo_shape field and not for geo_point.